### PR TITLE
Document known discoverable zha devices

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Document known discoverable zha devices

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/54935
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/zha.markdown [hunk 1]
```diff
@@ -145,13 +145,29 @@ A list of available device paths can be found in **Supervisor** > **System** > *
...
 - PiZigate : `pizigate:/dev/ttyS0`
 - Wifi Zigate : `socket://[IP]:[PORT]` for example `socket://192.168.1.10:9999`
 - Sonoff ZBBridge : `socket://[IP]:[PORT]` for example `socket://192.168.1.11:8888`
 
+### Discovery via USB or Zeroconf
+
+Some devices can be auto-discovered, which can simplify the ZHA setup process. The following devices have been tested with discovery, and offer a quick setup experience:
+
+| Device | Discovery Method | Identifier | Vendor |
+| -------| ---------------- | ------ |
```
Comment (bdraco) [inline source/_integrations/zha.markdown]:
```suggestion
| Device | Discovery Method | Identifier | Vendor |
| -------| ---------------- | ---------- | ------ |
```

File: source/_integrations/zha.markdown [hunk 2]
```diff
@@ -145,13 +145,29 @@ A list of available device paths can be found in **Supervisor** > **System** > *
...
+
+| Device | Discovery Method | Identifier | Vendor |
+| -------| ---------------- | ---------- | ------ |
+| ConBee II | USB | 1CF1:0030 | https://phoscon.de/en/conbee2 |
+| Electrolama zig-a-zig-ah | USB | 1A86:7523 | https://electrolama.com/projects/zig-a-zig-ah/ |
+| Nortek HUSBZB-1 | USB | 10C4:8A2A | https://www.nortekcontrol.com/products/2gig/husbzb-1-gocontrol-quickstick-combo/ |
+| slae.sh CC2652RB development stick | USB | 10C4:EA60 | https://slae.sh/projects/cc2652/ |
+| Tube Zigbee Coordinator | Zeroconf | tube_zb_gw_cc2652p2.local. | https://www.tubeszb.com/ |
+
+Additional devices in the [Known working Zigbee radio modules](#known-working-zigbee-radio-modules) list may be discoverable, but have not been confirmed.
```
Comment (bdraco) [inline source/_integrations/zha.markdown]:
```suggestion
Additional devices in the [Known working Zigbee radio modules](#known-working-zigbee-radio-modules) list may be discoverable, however only devices that have been confirmed discoverable are listed above.
```

**Review by frenck — COMMENTED** [review]
Although a good addition to the documentation, my comment in the core codebase was related to the manifest, not documentation.

**Comment by bdraco** [comment]
> Although a good addition to the documentation, my comment in the core codebase was related to the manifest, not documentation.

Was totally confused as I didn't see your comment in the other PR until after I read this :)

I'll add a description field on the core side

File: source/_integrations/zha.markdown [hunk 3]
```diff
@@ -145,13 +145,29 @@ A list of available device paths can be found in **Supervisor** > **System** > *
...
 If you are use ZiGate or Sonoff ZBBridge you have to use some special usb_path configuration:
 
 - ZiGate USB TTL or DIN: `/dev/ttyUSB0` or `auto` to auto discover the zigate
 - PiZigate : `pizigate:/dev/ttyS0`
 - Wifi Zigate : `socket://[IP]:[PORT]` for example `socket://192.168.1.10:9999`
 - Sonoff ZBBridge : `socket://[IP]:[PORT]` for example `socket://192.168.1.11:8888`
 
+### Discovery via USB or Zeroconf
+
+Some devices can be auto-discovered, which can simplify the ZHA setup process. The following devices have been tested with discovery, and offer a quick setup experience:
```
Comment (frenck) [inline source/_integrations/zha.markdown]:
```suggestion
Some devices can be auto-discovered, which can simplify the ZHA setup process. The following devices have been tested with discovery, and offer a quick setup experience:
```
```suggestion
Some devices can be auto-discovered, which can simplify the ZHA setup process. The following devices have been tested with discovery and offer a quick setup experience:
```

File: source/_integrations/zha.markdown [hunk 4]
```diff
@@ -145,13 +145,29 @@ A list of available device paths can be found in **Supervisor** > **System** > *
...
+
+| Device | Discovery Method | Identifier | Vendor |
+| -------| ---------------- | ---------- | ------ |
+| ConBee II | USB | 1CF1:0030 | https://phoscon.de/en/conbee2 |
+| Electrolama zig-a-zig-ah | USB | 1A86:7523 | https://electrolama.com/projects/zig-a-zig-ah/ |
+| Nortek HUSBZB-1 | USB | 10C4:8A2A | https://www.nortekcontrol.com/products/2gig/husbzb-1-gocontrol-quickstick-combo/ |
+| slae.sh CC2652RB development stick | USB | 10C4:EA60 | https://slae.sh/projects/cc2652/ |
+| Tube Zigbee Coordinator | Zeroconf | tube_zb_gw_cc2652p2.local. | https://www.tubeszb.com/ |
```
Comment (frenck) [inline source/_integrations/zha.markdown]:
```suggestion
| Device | Discovery Method | Identifier |
| -------| ---------------- | ---------- |
| [ConBee II](https://phoscon.de/en/conbee2) | USB | 1CF1:0030 |
| [Electrolama zig-a-zig-ah](https://electrolama.com/projects/zig-a-zig-ah/) | USB | 1A86:7523 |
| [Nortek HUSBZB-1](https://www.nortekcontrol.com/products/2gig/husbzb-1-gocontrol-quickstick-combo/) | USB | 10C4:8A2A |
| [slae.sh CC2652RB development stick](https://slae.sh/projects/cc2652/) | USB | 10C4:EA60 |
| [Tube Zigbee Coordinator](https://www.tubeszb.com/) | Zeroconf | tube_zb_gw_cc2652p2.local. |
```

Additionally: Is discovery method really interesting/needed/relevant for the end-user documentation?

File: source/_integrations/zha.markdown [hunk 5]
```diff
@@ -145,13 +145,29 @@ A list of available device paths can be found in **Supervisor** > **System** > *
...
+
+Additional devices in the [Known working Zigbee radio modules](#known-working-zigbee-radio-modules) list may be discoverable, however only devices that have been confirmed discoverable are listed above.
```
Comment (frenck) [inline source/_integrations/zha.markdown]:
```suggestion
Additional devices in the [Known working Zigbee radio modules](#known-working-zigbee-radio-modules) list may be discoverable, however, only devices that have been confirmed discoverable are listed above.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/zha.markdown (hunk 4 above)
Comment (bdraco) [reply source/_integrations/zha.markdown]: That's a hard question to answer as it is interesting to me as I didn't realize until earlier this year there was support for communicating with a remote coordinator over a socket which caused me to switch all of my setups to use Tube's gear.

But I'm not likely the typical user... I highly value being able to restart production systems frequently (10-20x an hour) to test development changes.

Comment (frenck) [reply source/_integrations/zha.markdown]: Ok, lets keep it, it doesn't harm :)

**Review by frenck — APPROVED** [review]
Thanks, @bdraco 👍

**Pull request merged**